db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/UserTaskWriter.java

```diff
@@ -81,4 +82,23 @@ public void update(final UserTaskDbModel userTaskDbModel) {
               userTaskDbModel));
     }
   }
+
+  public void migrateToProcess(
+      final long userTaskKey,
+      final long processDefinitionKey,
+      final String processDefinitionId,
+      final int processDefinitionVersion,
+      final String elementId) {
+    executionQueue.executeInQueue(
+        new QueueItem(
+            ContextType.USER_TASK,
+            userTaskKey,
+            "io.camunda.db.rdbms.sql.UserTaskMapper.migrateToProcess",
+            Map.of(
```

Comment:
Hm, shouldn't we be consistent regarding how we pass data to mybatis? So by using a pojo ...